celavi/__main__.py

```diff
@@ -89,8 +91,12 @@
 
 turbine_data_filename = os.path.join(args.data, 'inputs', 'number_of_turbines.csv')
 
-
-data_filtering_choice = False
+#The code is setup such that data filtering is always true unless user sends 'US' as an argument to the code run statement. 
+#Sending any other arguments, namely state abbreviations will result in filtering to occur
+#Send python __main__.py --list US for national scale run
+#Send python __main__.py --list IO CO .... for state runs. 
+#maximum of 6 states can be entered on the HPC runs right now. Can be modified later. 
+data_filtering_choice = True
```

Comment:
I don't think over-writing an input parameter value by default is a good idea. The way the filtering will work when we have the config files up and running is that `data_filtering_choice` and the list of states to filter are both inputs (in the config file), and the filtering code only executes if `data_filtering_choice` is True. I think we should keep it that way, partially to avoid over-writing the `data_filtering_choice` and partially because having to send the 'US' flag will be misleading when running CELAVI on a dataset that doesn't need filtering but isn't national scale.

If this functionality currently works better on HPC, then as a stopgap measure until the config file functionality is merged into `develop` I recommend not merging this PR and continuing to execute HPC runs off this branch. I can prioritize completing the config file functionality and try to get that PR tested and merged into `develop` by tomorrow morning.

Reply:
> I don't think over-writing an input parameter value by default is a good idea. The way the filtering will work when we have the config files up and running is that `data_filtering_choice` and the list of states to filter are both inputs (in the config file), and the filtering code only executes if `data_filtering_choice` is True. I think we should keep it that way, partially to avoid over-writing the `data_filtering_choice` and partially because having to send the 'US' flag will be misleading when running CELAVI on a dataset that doesn't need filtering but isn't national scale.
> 
> If this functionality currently works better on HPC, then as a stopgap measure until the config file functionality is merged into `develop` I recommend not merging this PR and continuing to execute HPC runs off this branch. I can prioritize completing the config file functionality and try to get that PR tested and merged into `develop` by tomorrow morning.

Agreed.